.latexindent.yaml

```diff
@@ -0,0 +1,5 @@
+defaultIndent: "  "
+onlyOneBackUp: 1
+modifyLineBreaks:
+    textWrapOptions:
+        columns: 100
```

Comment:
This file contains the `latexindent` config (specifically what we want different from the default).
1. I changed it use 2 spaces instead of a single tab (`\t`) for indents.
2. Max line length is 100 characters. With the way it does formatting, it is possible for lines to be longer than 100 characters as it does multiple passes and does different updates each pass.